bootstrap/shared/management/main.tf

```diff
@@ -0,0 +1,69 @@
+#  Copyright (c) University College London Hospitals NHS Foundation Trust
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+variable "naming_suffix" {
+  type = string
+}
+
+variable "naming_suffix_truncated" {
+  type = string
+}
+
+variable "location" {
+  description = "The Azure region you wish to deploy resources to"
+  type        = string
+
+  validation {
+    condition     = can(regex("[a-z]+", var.location))
+    error_message = "Only lowercase letters allowed"
+  }
+}
+
+resource "azurerm_resource_group" "management" {
+  name     = "rg-mgmt-${var.naming_suffix}"
+  location = var.location
+}
+
+resource "azurerm_storage_account" "management" {
+  name                     = "stgmgmt${var.naming_suffix_truncated}"
+  resource_group_name      = azurerm_resource_group.management.name
+  location                 = azurerm_resource_group.management.location
+  account_tier             = "Standard"
+  account_replication_type = "LRS"
+}
+
+resource "azurerm_storage_container" "tfstate" {
+  name                 = "tfstate"
```

Comment:
does this clash if only one management storage account in CI?

Reply:
This is only created once for CI with `make ci`

Reply:
sorry, poor explanation: meant if the state is being named the same in the same container = clash?

Reply:
Right ok - so the state is now prefixed like so to prevent this: 
```
"${get_env("SUFFIX_OVERRIDE", dependency.bootstrap.outputs.environment)}/${path_relative_to_include()}/terraform.tfstate"
```